docs/adopters.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Adopters
-sidebar_position: 7
+sidebar_position: 8
```

Comment:
Can you make all these 70, 80, 90 etc so we don't have to change *all* of these every time.

Also, little PRs help with little things like this

Reply:
@TerryHowe for now, I have only removed the installation from the guides and put it at the top level. The position will update correctly after PR #191 merges.